ERROR:root:
Traceback (most recent call last):
  File "/Prosenjit/ProPythonProject/ProPythonTestPractice/WebScraping/MainWebScrapy.py", line 7, in <module>
    raise Exception
Exception
